src/applications/edu-benefits/hooks/useSetToggleParam.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+
+export const useSetToggleParam = toggleValue => {
+  useEffect(
+    () => {
+      const urlParams = new URLSearchParams(window.location.search);
+      if (
+        toggleValue !== undefined &&
+        toggleValue !== null &&
+        toggleValue !== 'undefined'
```

Comment:
Can this check just be !toggleValue?

Reply:
yeah sure will change that

Reply:
actually that cause kinda of infinite loop .page keeps loading